modules/_labs/badge/css/lib/badge.scss

```diff
@@ -0,0 +1,53 @@
+@keyframes wdc-badge-grow {
+  from {
+    transform: scale(0.85);
+  }
+
+  to {
+    transform: scale(1);
+  }
+}
+
+@mixin wdc-badge {
+  // base styles
+  align-items: center;
+  animation: wdc-badge-grow 0.2s ease;
+  border-radius: 999px;
+  box-sizing: border-box;
+  display: inline-flex;
+  font-family: $wdc-font-family;
+  font-size: 11px;
+  font-weight: 700;
+  height: 20px;
+  justify-content: center;
+  min-width: 20px;
+  padding: 0 6.5px;
+}
+
+// default badge styles
+.wdc-badge {
+  @include wdc-badge();
+  background: $wdc-color-cinnamon-500;
+  color: $wdc-color-french-vanilla-100;
+  text-shadow: 0 0 1px rgba(0, 0, 0, 0.35);
+
+  &:empty {
+    background: $wdc-color-french-vanilla-100;
```

Comment:
Weird visual thing, but this dot seems so much larger than the blue one 😂  (though I know it's not)

Reply:
Right! I saw that too! Visual trickery. I'm glad I'm not the only one. There wasn't a spec for this, so I'm going to run it by Rick. 👍